devstack/README.rst

```diff
@@ -0,0 +1,24 @@
+==========================
+Devstack plugin for Calico
+==========================
+
+1. Download devstack
+
+2. Add to your devstack local.conf file::
```

Comment:
I think it'd be worth explaining what to start with.  What if I've already got a local.conf with ovs configured?